<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningMainWindow2</class>
 <widget class="QMainWindow" name="WarningMainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>378</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>60</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>272</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>378</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>484</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>378</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>484</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mg/m3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>272</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>166</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>177</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>70</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>166</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>806</x>
      <y>71</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>60</y>
      <width>149</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>166</y>
      <width>149</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%rh</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_10">
    <property name="geometry">
     <rect>
      <x>806</x>
      <y>495</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>25.3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>495</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>85.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>806</x>
      <y>177</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>71</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>283</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>45.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>378</y>
      <width>149</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>272</y>
      <width>149</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>60</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>166</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>166</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>806</x>
      <y>389</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>79.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>484</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>粉尘大小</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>60</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>378</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>静电</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>484</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>272</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>光敏度</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>806</x>
      <y>283</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>45.7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>148</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>272</y>
      <width>148</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>590</y>
      <width>148</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>389</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>75.3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>484</y>
      <width>149</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mg/m3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
